src/functions.cpp

```diff
@@ -141,10 +141,28 @@ void removeFile(std::string_view filename)
     }
 }
 
+/**
+ * \brief Get the current timestamp as a string.
+ *
+ * This function generates the current timestamp in the format
+ * "YYYYMMDD_HHMMSS".
+ *
+ * \return A string with the current timestamp.
+ */
+std::string getCurrentTimestamp()
+{
+    time_t now;
+    time(&now);
+    char buf[sizeof "2011-10-08T07:07:09Z"];
+    strftime(buf, sizeof buf, "%FT%TZ", gmtime(&now));
```

Comment:
Yes, this can be done with `chrono`
https://en.cppreference.com/w/cpp/chrono

Reply:
Need to use stringstream if using chrono. Already discussed with @viniciusferrao

Reply:
The beautiful thing of C++ is the legacy that we have. There's probably a way to completely ditch `stringstream` because it sucks and we have `std::format` now, however we just need to find a way to use it with `chrono`.